internal/server/filesystem/internal/v1alpha1/conversion_generated.go

```diff
@@ -0,0 +1,34 @@
+// Code generated by csi-proxy-gen. DO NOT EDIT.
+
+package v1alpha1
+
+import (
+	"github.com/kubernetes-csi/csi-proxy/client/api"
+	pb "github.com/kubernetes-csi/csi-proxy/client/api/filesystem/v1alpha1"
+	"github.com/kubernetes-csi/csi-proxy/internal/server/filesystem/internal"
+)
+
+func autoconvert_pb_ComputeDoubleRequest_To_internal_ComputeDoubleRequest(in *pb.PathExistsRequest, out *internal.PathExistsRequest) error {
```

Comment:
Is this expected? The ComputeDouble stuff is from the dummy API group. They should not show up under the filesystem API group, correct?

Reply:
And that's why humans are bad at "auto-generating" code ;) good catch!